### DISCUSSION TITLE
Running Gauge without Powershell, is it possible?

### DISCUSSION THREAD

**Discussion by jkchua** [getgauge/gauge#2256] [General]
Good day everyone! :) 

Currently, I am building the base framework using the example spec. However, when I ran the example specification after creating the project, it seems like it's running PowerShell, which is currently restricted on our side. 

![image](https://user-images.githubusercontent.com/11019850/176580243-b14d4be0-da62-4b47-8c31-360b1d0b70dc.png)

I would like to ask here if is there a way for me to run Gauge specifications without using PowerShell on Windows platform or is it the only way to do it?

**Comment by sriv** [comment]
Can you try installing gauge using the windows installer or chocolatey, and not via npm? The npm installer uses a `ps1` script to launch gauge

**Reply by jkchua** [reply]
I'll give it a try. I think I found a way with using command prompt, but will try your recommendation. Thank you very much 😃

**Reply by jkchua** [reply]
Getting back to you - it worked! I should have just installed Gauge via windows installer than with npm. Thank you again 😊😊

**Reply by sriv** [reply]
Thanks for getting back! Glad you are set.
